README.md

```diff
@@ -459,6 +459,18 @@ It's possible to verify the operator status by running:
 $ oc describe operatorsource.operators.coreos.com/kogitocloud-operator -n openshift-marketplace
 ```
 
+#### Running End to End tests
+
+If you have an OpenShift cluster and admin privileges, you can run e2e tests with the following command:
+
+```bash
+$  ./hack/run-e2e.sh <namespace> <tag>
```

Comment:
I think we can create a task in the makefile for the e2e testing.

Reply:
I was thinking about that. I guess won't hurt adding to the Makefile